.travis.yml

```diff
@@ -46,12 +46,14 @@ matrix:
            CHECK_MBSTRING=0
   allow_failures:
     - php: "hhvm"
+    - php: "nightly"
```

Comment:
Let the `php-nightly` test be failed because nobody can guarantee that the unstable PHP version will be always successful.